src/PrestaShopBundle/Resources/config/services/core/form/data_formatter.yml

```diff
@@ -1,6 +1,10 @@
 services:
   _defaults:
-    public: true
+    autowire: true
+    bind:
+      $modifyAllNamePrefix: !php/const PrestaShopBundle\Form\Admin\Extension\ModifyAllShopsExtension::MODIFY_ALL_SHOPS_PREFIX
+
+  PrestaShop\PrestaShop\Core\Form\IdentifiableObject\DataFormatter\AbstractFormDataFormatter: ~
```

Comment:
I don't think you need this anymore since the construct parameter is already autowired You don't use this service as an abstract/parent service anymore so you should remove it